### PR TITLE
[FLINK-12722][docs] Adds Python Table API tutorial

### PR DESCRIPTION
**Pull request by dianfu**
## What is the purpose of the change

*This pull request adds a Python Table API tutorial to help beginners of Python Table API to get a basic knowledge of how to create a simple Python Table API job.*

## Brief change log

  - *Adds python_table_api.md and python_table_api.zh.md*

## Verifying this change

This change is a documentation work without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

File: docs/tutorials/python_table_api.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+Firstly, you can fire up your favorite IDE and create a Python project and then
+you need to install the PyFlink package. Please
+see [Build PyFlink]({{ site.baseurl }}/flinkDev/building.html#build-pyflink)
+for more details about this.
+
+## Writing a Flink Python Table API Program
+
+The first step in a Flink Python Table API program is to create a `BatchTableEnvironment`
+(or `StreamTableEnvironment` if you are writing a streaming job). It is the main entrypoint
```
Comment (sunjincheng121) [inline docs/tutorials/python_table_api.md]: `entrypoint` -> `entry point`?

File: docs/tutorials/python_table_api.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+exec_env = ExecutionEnvironment.get_execution_environment()
+exec_env.set_parallelism(1)
+t_config = TableConfig()
+t_env = BatchTableEnvironment.create(exec_env, t_config)
+{% endhighlight %}
+
+The `ExecutionEnvironment` (or `StreamExecutionEnvironment` if you are writing a streaming job)
+can be used to set execution parameters, such as the restart strategy, default parallelism, etc.
+
+The `TableConfig` can be used by set the parameters such as the built-in catalog name, the
```
Comment (sunjincheng121) [inline docs/tutorials/python_table_api.md]: `set`->`setting`?

File: docs/tutorials/python_table_api.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+threshold where generating code, etc.
+
+Next we will create a source table and a sink table.
+
+{% highlight python %}
+t_env.connect(FileSystem().path('/tmp/input')) \
+    .with_format(OldCsv()
+                 .line_delimiter(' ')
+                 .field('word', DataTypes.STRING()))\
```
Comment (sunjincheng121) [inline docs/tutorials/python_table_api.md]: Add a space for `\`

File: docs/tutorials/python_table_api.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+exec_env = ExecutionEnvironment.get_execution_environment()
+exec_env.set_parallelism(1)
+t_config = TableConfig()
+t_env = BatchTableEnvironment.create(exec_env, t_config)
+
+t_env.connect(FileSystem().path('/tmp/input')) \
+    .with_format(OldCsv()
+                 .line_delimiter(' ')
+                 .field('word', DataTypes.STRING()))\
```
Comment (WeiZhong94) [inline docs/tutorials/python_table_api.md]: Add a space between ")" and "\"?

File: docs/tutorials/python_table_api.zh.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+{% highlight bash %}
+$ python WordCount.py
+{% endhighlight %}
+
+上述命令会构建Python Table API程序，并在本地mini cluster中运行。如果想将作业提交到远端集群执行，
+可以参考[作业提交示例]({{ site.baseurl }}/zh/ops/cli.html#job-submission-examples)。
+
+上述教程介绍了如何编写并运行一个Flink Python Table API程序，如果想了解Flink Python Table API
+的更多信息，可以参考[Flink Python Table API文档]({{ site.pythondocs_baseurl }}zh//api/python)。
```
Comment (WeiZhong94) [inline docs/tutorials/python_table_api.zh.md]: '//' -> '/'

**Comment by dianfu** [comment]
@sunjincheng121 @WeiZhong94 Thanks a lot for the review. Updated accordingly.

**Comment by sunjincheng121** [comment]
@flinkbot approve all
+1 to merged

**Pull request closed without merging**